### ISSUE TITLE
Bug: BulkInsert doesn't use PropertyHandler

### ISSUE THREAD

**Issue by Garios** [mikependon/RepoDb#1008]
### Bug Description
I wrote a PropertyHandler to convert TimeSpan-(Ticks) to (db-)bigint.

On InsertAsync() PropertyHandlers works fine.
On BulkInsertAsync() PropertyHandlers aren't used.

```csharp
FluentMapper.Entity<TABLE>().PropertyHandler<PropertyHandlerTimeSpan>(p => p.Duration);
```

Example: RepoDb v1.12.9 (also 1.12.10-beta4) and RepoDb.SqlServer v1.1.4 (1.1.5-beta3) and SqlServer.BulkOperations 1.1.5 (also 1.1.6-beta1)

**Comment by mikependon** [comment]
Somebody place this issue on our RepoDB.NET site.
Referencing: https://github.com/mikependon/RepoDB.NET/issues/41

**Comment by mikependon** [comment]
Yes, this is not triggered on the Bulk Operations yet (both SQL Server and PostgreSQL). This is on our timeline.

**Comment by Garios** [comment]
oh, I didn't realized that there is another repo. Thank you!

**Comment by mikependon** [comment]
This is the right place, the other repo is for the documentation site.

**Comment by ghost** [comment]
Would be great to have this enhancement. I ran into it a few moments ago.
